ci/test/mkpipeline.py

```diff
@@ -139,6 +139,22 @@ def inputs(self) -> Set[str]:
         return inputs
 
 
+def prioritize_pipeline(pipeline: Any) -> None:
+    """Prioritize builds against main or release branches"""
+
+    branch = os.environ["BUILDKITE_BRANCH"]
+    priority = None
+
+    if branch.startswith("v"):
+        priority = 10
```

Comment:
```suggestion
        priority += 10
```

☝🏽 will keep the relative priority of jobs, which seems nice.

Reply:
@philip-stoev I don't think this change got applied?

Reply:
Oh, no, sorry, I just gave you a terrible suggestion. Nvm. I see it below.